kpi/serializers/current_user.py

```diff
@@ -98,6 +103,14 @@ def get_validated_password(self, obj):
 
         return extra_details.validated_password
 
+    def get_accepted_tos(self, obj: User) -> bool:
+        try:
```

Comment:
Add a docstring here just explaining what this does. It may not be obvious that this is a simplified and private property on the API. Spell out terms of service in these docs too.